SharpinitUnitTest/UnitRegistryTests.cs

```diff
@@ -4,20 +4,33 @@
 using SharpInit.Units;
 using System;
 using System.IO;
+using System.Text;
 
 namespace UnitTests.Units
 {
     [TestClass]
     public class UnitRegistryTests
     {
 
+        static string path = ".\\test.service";
+        static string contents = "[Unit]\nDescription = Notepad\n\n[Service]\nExecStart = notepad.exe";
```

Comment:
Maybe this could be structured like so?

    static string Contents = "[Unit]\n" + 
        "Description = Notepad\n" +
        "\n" + 
        "[Service]\n" + 
        "ExecStart = notepad.exe";

which VS supports formatting into natively, just hit Enter after the newlines

Reply:
Done